gpustat/__main__.py

```diff
@@ -12,20 +12,26 @@ def print_gpustat(json=False, debug=False, **args):
     '''
     Display the GPU query results into standard output.
     '''
-    try:
-        gpu_stats = GPUStatCollection.new_query()
-    except Exception as e:
-        sys.stderr.write('Error on querying NVIDIA devices. Use --debug flag for details\n')
-        if debug:
-            import traceback
-            traceback.print_exc(file=sys.stderr)
-        sys.exit(1)
-
-    if json:
-        gpu_stats.print_json(sys.stdout)
-    else:
-        gpu_stats.print_formatted(sys.stdout, **args)
-
+    isloop = args.pop('loop')
```

Comment:
A `noloop` may be meaningful.

Reply:
agree, I was thinking that `store_false` defaults to False...